### PR TITLE
style(lib/core): var -> let and const codemod

### PR DESCRIPTION
**Pull request by gaiety-deque**
Just like https://github.com/dequelabs/axe-core/pull/4451 but for `lib/core`

This does add an `.eslintignore` line, perhaps we'd rather skip a file a different way.

Smaller part of the full vision: https://github.com/dequelabs/axe-core/pull/4444

### CODE REVIEW

**Review by romankulkovsf — APPROVED** [review]

File: .eslintignore [hunk 1]
```diff
@@ -7,5 +7,6 @@ doc/api/*
 doc/examples/jest_react/*.js
 
 lib/core/imports/*.js
+lib/core/utils/uuid.js
```
Comment (WilcoFiers) [inline .eslintignore:10]: Did you mean to include this?

**Review by WilcoFiers — CHANGES_REQUESTED** [review]
the eslintignore thing. Rest looks good

File: .eslintignore (hunk 1 above)
Comment (gaiety-deque) [reply .eslintignore:10]: I did @WilcoFiers. It felt weird to update a file that I don't think we wrote (judging by the comment at the top of `uuid.js`. Eventually when all of these codemods are merged the last PR will turn on the eslint rule `no-var` and unless we either update this file or ignore it, this file will throw an eslint warning.

Comment (WilcoFiers) [reply .eslintignore:10]: @gaiety-deque Merge conflict, can you update?

**Review by WilcoFiers — APPROVED** [review]

**Pull request merged**